proxy/attempt-register-vm-on-proxy.sh

```diff
@@ -73,8 +79,13 @@ echo "Backend id: ${BACKEND_ID}"
 
 # Store the registration information in a ConfigMap
 kubectl create configmap inverse-proxy-config \
+        --namespace=${NAMESPACE}
         --from-literal=ProxyUrl=${PROXY_URL} \
         --from-literal=BackendId=${BACKEND_ID} \
         --from-literal=Hostname=${HOSTNAME}
 
+if [ ! -z "${K8S_APP_NAME}" ]; then
+    kubectl label configmap inverse-proxy-config app.kubernetes.io/name=${K8S_APP_NAME}
```

Comment:
have you verify GC would work? 
I think the configmap need owner reference. E.g. 
```
 ownerReferences:
  - apiVersion: app.k8s.io/v1beta1
    blockOwnerDeletion: true
    kind: Application
    name: mk-3
    uid: 2a108aa3-d7a3-11e9-aebe-42010a80009c
```

Reply:
Per mil (cc'ed you), it uses ownerRef to delete it, not label selection. So this PR can't work. 
Unfortunately "kubectl create configmap" doesn't have ownerRef parameter. Please ignore this PR for now. Didn't find a way yet.

Reply:
An alternative way is to store the proxy URL as the proxy agent deployment's label, instead of  storing it as a separate configmap. This way we end up not needed to create a new resource. The deployment will be deleted. 

https://github.com/kubeflow/pipelines/blob/master/manifests/gcp_marketplace/chart/kubeflow-pipelines/templates/proxy.yaml#L42

Reply:
or updating a pre-created configmap which is more easy as if we use label, we will need new API to query it. not a small sized work in our landing page.

I will update this PR later.

Reply:
Updating a pre-created configmap SGTM